### PR TITLE
Partner Highlight hidora

### PR DESCRIPTION
**Pull request by piellick**
### Description
[proposal about Highlighting hidora partner]
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/partners-hidora [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Partner Highlight: Enjoy OpenSearch one click installation with Hidora
```
Comment (stockholmux) [inline _posts/partners-hidora:1]: Can you add the front-matter?   ([You can see how other posts are structured](https://github.com/opensearch-project/project-website/blob/main/_posts/2022-01-21-dashboards-plugins-intro.markdown)).

Example:
```
---
layout: post
title: "Partner Highlight: Enjoy OpenSearch one click installation with Hidora"
authors:
  - piellick 
date: 2022-01-21
categories:
  - partners
twittercard:
  description: "Hidora is an expert in providing custom solutions for all sizes of businesses. Hidora Cloud Hosting Solutions are just what you need if you want an efficient, secure and scalable cloud hosting solution that is tailored to your needs. We offer different plans that are scalable according to your specific requirements so you can find one that suits you best. Our PaaS help developers to host their applications in few minutes with DevOps philosophy."
---
```

Comment (stockholmux) [reply _posts/partners-hidora:1]: Also, you need an author bio in another file ([example](https://github.com/opensearch-project/project-website/blob/main/_authors/andhopp.markdown))

File: _posts/partners-hidora [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+*   Activate or not OpenSearch-Dashboard and Logstash
+*   Define how many nodes will be create on OpenSearch cluster
+*   Assign public IPv4 or IPv6 or manage access through our Shared Load Balancer (SLB)
+*   Scale nodes resources by managing in real time Cloudlet assignation
+
+  
+
+After being created, the OpenSearch node is available on port _9200_ for internal communication between nodes of the environment and port _4848_ for external services through the shared load balancer. The HTTP basic auth is used in both cases (password can be reset using the **Reset Password** button).
+
+![](https:///image.png)
```
Comment (stockholmux) [inline _posts/partners-hidora:33]: - Can you merge this with #633 so we can see your photos in place?
- Each image needs a `{: .img-fluid}` at the end of the line. That will make it work on mobile

**Review by stockholmux — CHANGES_REQUESTED** [review]
Hey this looks pretty good but it's pretty hard to review since it's in two PRs. Can you merge them? 

Also you'll need to fix your DCO sign off.

Glad to assist with these things and get this post going - the content looks great!

**Comment by piellick** [comment]
Hello @stockholmux , i close this one too .

**Pull request closed without merging**